server/server.js

```diff
@@ -0,0 +1,42 @@
+import dotenv from "dotenv";
+import { Configuration, OpenAIApi } from "openai";
+
+dotenv.config();
+const config = new Configuration({
+  // apiKey: process.env.OPEN_AI_API_KEY
+  apiKey: "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"
```

Comment:
dont ever include keys inside your code. it should be loaded from .env files, and the .env files should never be uploaded to github - make sure it's in git ignore.